components/library/tntnet/Solaris/http-tntnet.xml

```diff
@@ -0,0 +1,183 @@
+<?xml version="1.0"?>
+<!--
+CDDL HEADER START
+
+This file and its contents are supplied under the terms of the
+Common Development and Distribution License ("CDDL"), version 1.0.
+You may only use this file in accordance with the terms of version
+1.0 of the CDDL.
+
+A full copy of the text of the CDDL should have accompanied this
+source.  A copy of the CDDL is also available via the Internet at
+http://www.illumos.org/license/CDDL.
+
+CDDL HEADER END
+-->
+
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+    Copyright (c) 2007, 2011, Oracle and/or its affiliates. All rights reserved.
+    Copyright 2011 EveryCity Ltd. All rights reserved.
+    Copyright (c) 2016 Jim Klimov
+-->
+
+<service_bundle type='manifest' name='tntnet'>
+
+<service
+	name='network/http/tntnet'
```

Comment:
FWIW, I considered that commonality with apache2x, tomcat and lighttpd is
nice, but with our example usecase at work it is not unlikely that there
would be several copies running to serve different webapps. I guess it is a
matter of taste to call them http/tntnet:instX or http:tntnet-instX.

--
Typos courtesy of GMail on my Samsung Android
24.03.2016 16:09 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/library/tntnet/Solaris/http-tntnet.xml
> <https://github.com/OpenIndiana/oi-userland/pull/1777#discussion_r57331518>
> :
>
> > +http://www.illumos.org/license/CDDL.
> > +
> > +CDDL HEADER END
> > +-->
> > +
> > +<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
> > +<!--
> > +    Copyright (c) 2007, 2011, Oracle and/or its affiliates. All rights reserved.
> > +    Copyright 2011 EveryCity Ltd. All rights reserved.
> > +    Copyright (c) 2016 Jim Klimov
> > +-->
> > +
> > +<service_bundle type='manifest' name='tntnet'>
> > +
> > +<service
> > +	name='network/http/tntnet'
>
> Please, name it network/http and call instance tntnet to be consistent
> with other http services (nginx comes from EC and was a bad example)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1777/files/f3f31f4d279e4b02b87ceb5fb92b8a68c26a139a..cc739bc398ec10c740b0d3937f6723360fc64c71#r57331518>
>